lib/ansible/modules/system/ufw.py

```diff
@@ -245,10 +245,11 @@ def execute(cmd):
         cmd = ' '.join(map(itemgetter(-1), filter(itemgetter(0), cmd)))
 
         cmds.append(cmd)
-        (rc, out, err) = module.run_command(cmd)
+        (rc, out, err) = module.run_command(cmd, environ_update={"LANG": "en_EN"})
```

Comment:
Do you mean `en_US`? I don't think the locale `en_EN` actually exists.

Reply:
You're right, my current locale is1 fr_FR I did a quick and wrong transposition to en. I'll change that to en_US in my next commit.